### ISSUE TITLE
After upgrade to 10.7.0 RC3 jellyfin loads indefinitely

### ISSUE THREAD

**Issue by Adreotice** [jellyfin/jellyfin#5113]
So I am on Windows 10 and upgraded my jellyfin 10.6.4 to 10.7.0 RC3 using the installer and when I try to open it it just loads and loads. I waited 2 hours and nothing happened. I reinstalled many times and nothing.

**Comment by nyanmisaka** [comment]
Try to delete all plugins in jellyfin data folder and relaunch again.

**Comment by Adreotice** [comment]
> Try to delete all plugins in jellyfin data folder and relaunch again.

i deleted them but nothing's changed.

**Comment by nyanmisaka** [comment]
Try to clean the browser cache.

**Comment by Adreotice** [comment]
> Try to clean the browser cache.

That doesn't work either. I even tried other browsers

**Comment by Adreotice** [comment]
> Try to clean the browser cache.

I did a clean reinstall without saving metadata and it works now. It's just a pain that it has to search everything new. Good thing i added trakt plugin coule Weeks ago to save my viewed shows

**Comment by BaronGreenback** [comment]
Did you keep any log files?

**Comment by Adreotice** [comment]
> Did you keep any log files?

I didn't

**Comment by Rorschach1010** [comment]
This is coming late, I know but I think I have stumbled upon the same or a similar issue while switching between stable and latest branch with docker. Deleting the whole data folder was successful but after some little testing, I found out that I had to logout the user from testing before I can reload stable. This definitely works since I took the time to test it a few times. However, I'm not sure your problem is really the same.
